### DISCUSSION TITLE
combine and split books

### DISCUSSION THREAD

**Discussion by emacsomancer** [PaulWoitaschek/Voice#1366] [Ideas]
The (proprietary) Listen audiobook player allows separate files to be combined into a single book/track (this makes navigation much easier). And also allows for splitting of incorrectly combined files/directories. (Voice sometimes groups subdirectories together when they should not be.)

App-Version: 3.6.1
Phone Name: Samsung Galaxy S5
Android Version: LineageOS 14.1

**Comment by IRlyLikeBac0n** [comment]
Bump on this. Many audiobooks are separated into multiple files for easier uploading/downloading, so it would be helpful to either add a queue or option to combine/merge multiple files into one.
